### ISSUE TITLE
Returning Error with No Details

### ISSUE THREAD

**Issue by mudetroit** [plus3network/gulp-less#121]
I am getting an error out of gulp-less, but as you can see I am not receiving any details from it, making it difficult for me to pinpoint what the issue is..
```
[11:57:04] Starting 'styles'...
[11:57:06] Plumber found unhandled error:
 Error in plugin 'gulp-less'
Message:
    First argument needs to be a number, array or string. in file undefined line no. undefined
Details:
    lineNumber: undefined
    fileName: undefined
[11:57:07] Finished 'styles' after 3.02 s
[11:57:07] Starting 'watch'...
[11:57:07] Finished 'watch' after 155 ms
[11:57:07] Starting 'clientBuild'...
[11:57:07] Finished 'clientBuild' after 5.1 μs
```

Here is the styles task that runs gul less, it is coffeescript, but should be legible.
```
gulp = require 'gulp'
{less, sourcemaps, plumber} = require('gulp-load-plugins')()

gulp.task 'styles', ->
	gulp.src(['./public/include/less/**/*.less', '!./**/_*.less'])
	.pipe plumber()
	.pipe sourcemaps.init()
	.pipe less({verbose: true})
	.pipe sourcemaps.write()
	.pipe gulp.dest('./public/include/css')
```
When I remove sourcemaps from the equation the error goes away. I am not certain if it is being generated by gulp-less or gulp-sourcemaps, but something in the interaction of the two is causing an odd error. I will cross post this on gulp-sourcemaps as well.

floridoo/gulp-sourcemaps#78

**Comment by stevelacy** [comment]
Please create a test repo to reproduce this bug

**Comment by contra** [comment]
Hmm weird... it's possible that those attributes on the error coming out of `less` have some magic logic on them, maybe they are getters pointing to the sourcemap which gulp-sourcemap is modifying.

**Comment by luin** [comment]
same issue here

**Comment by arafati** [comment]
I ran into the same problem. I started debugging and found out that the bug resides in `less` module (specifically when it tries to create a base64 format for an `undefined` input).

In my case, I have a `variables.less` file that always ends up as an empty `variables.css` file that could not be handled by the sourcemap. So I just exclude it from the stream and everything went alright. I see you are excluding `_*.less` files but maybe you have another empty file somewhere.

**Comment by HaNdTriX** [comment]
https://github.com/plus3network/gulp-less/issues/122

**Comment by OhJeez** [comment]
Related (I hope): less/less.js#2430

**Comment by contra** [comment]
fixed in 2.0.3
